src/clover/load/load.py

```diff
@@ -798,14 +809,35 @@ def process_device_hourly_usage(
 
         logger.info("Hourly usage profile for %s successfully calculated.", device.name)
 
+        #Retrieve device laod time
+        n = device.load_time
+        
+        #Calculate cumulative load profile
+        extended_device_usage = {
+            column: [
+                hourly_device_usage[column].iloc[:i+1].sum() if i < int(n) else (
+                    hourly_device_usage[column].iloc[i-n+1:i+1].sum() if isinstance(n, int) else (
+                        hourly_device_usage[column].iloc[int(i-n+1)] * (1 - (i - n - int(i - n + 1) + 1)) + hourly_device_usage[column].iloc[int(i - n + 1) + 1:i + 1].sum()
+                    )
+            )
+            for i in range(len(hourly_device_usage))
+        ]
+        for column in hourly_device_usage.columns
+        }
+
+        #Concert to numeric so that analysis.py van use it
+        extended_device_usage = pd.DataFrame(extended_device_usage, index=hourly_device_usage.index)
+        extended_device_usage = extended_device_usage.apply(pd.to_numeric, errors='coerce')
```

Comment:
## ❓ Why this line?
What's this `erorrs='coerce'` line doing? Anything involving errors usually raises a 🚩 (red flag) as it's odd that any errors are arising in the first place in the above logic?